src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -783,15 +871,33 @@ def _add_optional_tags_to_vllm_json(
         index: int,
         add_model_name: bool,
         add_stream: bool,
+        extra_inputs: Dict,
+        output_tokens_mean,
+        output_tokens_stddev,
         model_name: str = "",
-        extra_inputs: Dict = {},
     ) -> Dict:
+        row = pa_json["data"][index]
         if add_model_name:
-            pa_json["data"][index]["model"] = model_name
+            row["model"] = model_name
         if add_stream:
-            pa_json["data"][index]["stream"] = [True]
+            row["stream"] = [True]
+        if output_tokens_mean != LlmInputs.DEFAULT_OUTPUT_TOKENS_MEAN:
+            number_of_tokens = str(
+                int(max(0, random.gauss(output_tokens_mean, output_tokens_stddev)))
+            )
+            sampling_parameters = {
```

Comment:
does this work for every backend/endpoint we currently support?

Reply:
No. This is the function for creating the JSON for the vLLM backend (_add_optional_tags_to_vllm_json). Each backend/endpoint has different requirements.